Add user mapping instructions (#69)
cookpad/cp8: README.md

project_column_id: 49 # Automatically add new issues to a project column
```

## User Mapping

> Your Github username needs to be mapped to your Slack ID so cp8 can send you slack notifications:

Reviewer: ```suggestion
Your GitHub username needs to be mapped to your Slack ID in order for CP8 to mention you:
```
